presto-main/src/main/java/io/prestosql/operator/StreamingAggregationOperator.java

```diff
@@ -299,7 +299,7 @@ private void processInput(Page page)
                     startPosition = nextGroupStart;
                 }
                 else {
-                    currentGroup = page.getRegion(page.getPositionCount() - 1, 1);
+                    currentGroup = page.getRegion(page.getPositionCount() - 1, 1).getLoadedPage();
```

Comment:
Maybe add a comment on why this is important, so future readers know why this is here.